src/clr-addons/components.clr-addons.scss

```diff
@@ -347,3 +370,11 @@ input:not([type='time']) {
   text-overflow: ellipsis;
   white-space: nowrap;
 }
+
+@media screen and (max-width: map-get($clr-grid-breakpoints, sm)) {
+  .main-container .content-container .content-area,
+  .content-area-footer,
```

Comment:
for .content-area-footer this will actually increases top/bottom margin (from .25rem). is this intended?

Reply:
This is intended, as the padding is needed for the users on smartphones to feel more comfortable.